ComponentKit/Utilities/CKComponentActionInternal.h

```diff
@@ -65,7 +66,7 @@ class CKTypedComponentActionBase {
   // that runs code on destruction, making this field the first field of this
   // object saves an offset calculation instruction in the destructor.
   __weak id _target;
-  CKScopedResponder *_scopedResponder;
+  std::pair<CKScopedResponderUniqueIdentifier, CKResponderGenerationBlock> _scopeIDAndGenerator;
```

Comment:
`_scopeIDAndGenerator` isn't terribly descriptive. Maybe `_scopeIdentifierAndResponderGenerator`? I dropped the ID since I'm not really a fan of the shorthand.`